game.html

```diff
@@ -19,9 +19,11 @@
         <script src="scripts/components/adjust-player-position.js"></script>
 
         <title> The Slide VR </title>
-
+        <audio autoplay loop>
+            <source src="http://www.gov.ru/main/symbols/gimn_rf_t2.mp3" type="audio/mpeg">
```

Comment:
i think we cannot use that song due to the licence

Reply:
Dude its is not even any licence it is literally a national anthem which author of died in 1946 its is literally public domain what are you talking about

Reply:
as far as i know this file is licenced Creative Commons Attribution 4.0 so you are free to distribute and modify the file as long as you attribute gov.ru site